README.md

```diff
@@ -1,56 +1,90 @@
 # Git Repository Basics
 
+
 ## Objectives
 
 1. Define Git and explain how it helps programmers control and manage changes
-2. Initialize a new Git repository with `git init`
-3. Check the state of your files using `git status`
-4. Stage files to commit using `git add` and `git commit`
-
-So, how do you use Git?
+2. State initializing Git repository with `git init`
+3. Use `git status` to check the status of your repository 
+4. Recognize how to keep track of files with `git add`
+5. Use `git commit` to apply a message to the commit
 
-## Creating a Repository with `git init`
+## Problem Statement
 
-You want to get started on a project that's going to be the next big thing. The first step is to make a new directory on your computer, probably in your development directory. To do so in the terminal, type:
+Have you ever found yourself moving between different workstations, or having to 
+work together in a team setting on the same sets of files? Your filenames get very
+verbose at times: 
 
 ```
-~ $ mkdir next-big-thing
-~ $ cd next-big-thing
-next-big-thing $ ls
-
+important_presentation-v1.docx
+important_presentation-v2.docx
+important_presentation-v3.docx
+important_presentation-final.docx
+important_presentation-final-final.docx
+important_presentation-final-final-FINAL.docx
+important_presentation-final-final-FINAL-SERIOUSLY.docx
 ```
+Luckily, there's a better way to keep track of versions of files and share them amongst 
+multiple parties without causing complete chaos or confusion.
 
-You created a new directory with `mkdir next-big-thing` and then moved into it by changing your directory with `cd next-big-thing`. You typed `ls` to see all the files and folders in your brand new project, and, as expected, it was empty. But this directory, `next-big-thing`, is where you're going to put all your code.
+## Define Git and explain how it helps programmers control and manage changes
 
-The first step is to transform this directory into a Git-enabled directory, which will allow it to keep track of all the changes to your code and interact with other Git remotes such as GitHub. How do we accomplish this? It's as simple as `git init`.
+If you have a project that you've already started building on your local machine, 
+you will want to be able to start tracking your changes and storing these changes 
+somewhere accessible so that other developers can review or collaborate on the project--
+or just put it somewhere you can come back to it later, regardless of what may happen 
+to the physical files on your computer. This is where `git` comes in! `Git` will allow 
+the users to keep track of these changes and keep detailed tabs and backups of your project.
```

Comment:
I think the "This is where..."  section goes in the last paragraph of the previous section. 

I don't think the first sentence before that go here. "tracking your changes" is a technical term.....I think a more natural point of entry is starting from a blank directory (as you do below). I'm inclined to cut that first sentence.

Reply:
And do we ever explain what "track changes" means in this doc? Seems like an appropriate thing to introduce, but it needs to be called out as a new vocabulary word.